src/main/java/io/confluent/connect/jdbc/source/TimestampHelper.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.connect.jdbc.source;
+
+import java.sql.Timestamp;
+import org.apache.kafka.connect.data.Struct;
+
+/**
+ * TimestampHelper is an interface to construct and operate on timestamp based queries. The
+ * implementation handles two types of queries: single timestamp or multiple timestamps.
+ */
+public interface TimestampHelper {
+
+  public void addWhereClause(StringBuilder builder, String quoteString, String incrementingColumn);
```

Comment:
This is pretty minor, but it'd be nice to have JavaDoc for these methods to be clear about expectations and constraints, such as whether `incrementingColumn` (or any of the other parameters) might ever be null.

Reply:
Makes sense. I meant to add that and then slipped through.